docs/DukeTest.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE HTML>
```

Comment:
May I know how you generated these files inside docs folder? I also checked other people's PR but the docs folder are all almost empty.

Reply:
I used Idea Intelijj's inbuild Javadoc tools to create it. It's not updated though.